### PR TITLE
Add conf-autoconf

### PR DESCRIPTION
**Pull request by wokalski**
I have no idea what I should put in the build section.

### CODE REVIEW

**Comment by ulrikstrid** [comment]
I think this is fine, we have something similar for gmp. @prometheansacrifice what do you think?

File: packages/conf-autoconf.0.1/package.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "build": [ [ "which", "autoconf"] ],
```
Comment (ManasJayanth) [inline packages/conf-autoconf.0.1/package.json]: Let's set it to 'true' instead

**Review by ManasJayanth — COMMENTED** [review]
This is great!

File: packages/conf-autoconf.0.1/package.json (hunk 1 above)
Comment (wokalski) [reply packages/conf-autoconf.0.1/package.json]: U mean `"build": true`? what does it mean?

**Comment by ManasJayanth** [comment]
It's a noop - a command that just exits with zero

**Comment by ManasJayanth** [comment]
Oh, I mean the bash command 'true'

**Comment by wokalski** [comment]
Done.

**Pull request merged**